war/package.json

```diff
@@ -29,7 +29,7 @@
     "copy-webpack-plugin": "^5.1.1",
     "css-loader": "^3.2.0",
     "css-minimizer-webpack-plugin": "^1.0.0",
-    "eslint": "^7.1.0",
```

Comment:
I've had to update `eslint` due to my use of optional chaining (e.g. `moreOptionsAllButton?.addEventListener`)